src/sequence/cache/strategies/default.js

```diff
@@ -0,0 +1,46 @@
+const { assign, omit } = require("lodash");
+
+const { sequence: makeNewSequence } = require("../../sequence");
+const { SequenceCache } = require("../sequence-cache");
+
+
+const cache = Object.create(null);
```

Comment:
Captured potential for enhancement here and out-of-band conversation: https://github.com/FormidableLabs/rapscallion/issues/37